ansible/inventories/devnet-9/group_vars/all/all.yaml

```diff
@@ -0,0 +1,185 @@
+#  ██║░░██║╚█████╔╝███████╗███████╗  ░░╚██╔╝░░██║░░██║██║░░██║██████╔╝
+#  ╚═╝░░╚═╝░╚════╝░╚══════╝╚══════╝  ░░░╚═╝░░░╚═╝░░╚═╝╚═╝░░╚═╝╚═════╝░
+
+# role: ethpandaops.general.ethereum_genesis
+ethereum_genesis_generator_container_image: "ethpandaops/ethereum-genesis-generator:1.3.4"
+ethereum_genesis_generator_version: "{{ ethereum_genesis_generator_container_image.split(':')[-1] }}"
+ethereum_genesis_network_seed: "{{ ansible_inventory_sources[0] }}"
+ethereum_genesis_chain_id: 7011893059
+ethereum_genesis_generator_output_dir: "../network-configs/{{ ethereum_network_name | replace('dencun-', '') }}"
+ethereum_genesis_timestamp: 1695981540
+ethereum_genesis_timedelay: 60
+ethereum_genesis_timestamp_relative_cmd:
+  Linux: "date +%s -d '+45 minutes'"
+  Darwin: "date -v +30M +%s"
+ethereum_genesis_mnemonic: "{{ secret_genesis_mnemonic }}"
+ethereum_genesis_generator_config_files:
+  cl/config.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/config.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  cl/mnemonics.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/mnemonics.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  el/genesis-config.yaml: |
+    chain_id: ${CHAIN_ID}
+    deposit_contract_address: "${DEPOSIT_CONTRACT_ADDRESS}"
+    mnemonic: ${EL_AND_CL_MNEMONIC}
+    el_premine:
+      "m/44'/60'/0'/0/0": 10000000000ETH
+      "m/44'/60'/0'/0/1": 10000000000ETH
+      "m/44'/60'/0'/0/2": 10000000000ETH
+      "m/44'/60'/0'/0/3": 10000000000ETH
+      "m/44'/60'/0'/0/4": 10000000000ETH
+      "m/44'/60'/0'/0/5": 10000000000ETH
+    el_premine_addrs: {}
+    genesis_timestamp: ${GENESIS_TIMESTAMP}
+    genesis_delay: ${GENESIS_DELAY}
+    slot_duration_in_seconds: ${SLOT_DURATION_IN_SECONDS}
+    capella_fork_epoch: ${CAPELLA_FORK_EPOCH}
+    deneb_fork_epoch: ${DENEB_FORK_EPOCH}
+
+  values.env: |-
+    export CHAIN_ID="{{ ethereum_genesis_chain_id }}"
+    export DEPOSIT_CONTRACT_ADDRESS="0x6f22fFbC56eFF051aECF839396DD1eD9aD6BBA9D"
+    export EL_AND_CL_MNEMONIC="{{ ethereum_genesis_mnemonic }}"
+    export CL_EXEC_BLOCK="0"
+    export DEPOSIT_CONTRACT_BLOCK="0x0000000000000000000000000000000000000000000000000000000000000000"
+    export NUMBER_OF_VALIDATORS=146
+    export GENESIS_FORK_VERSION="0x10{{ ethereum_genesis_fork_version_suffix }}"
+    export ALTAIR_FORK_VERSION="0x20{{ ethereum_genesis_fork_version_suffix }}"
+    export BELLATRIX_FORK_VERSION="0x30{{ ethereum_genesis_fork_version_suffix }}"
+    export CAPELLA_FORK_VERSION="0x40{{ ethereum_genesis_fork_version_suffix }}"
+    export CAPELLA_FORK_EPOCH="5"
+    export DENEB_FORK_VERSION="0x50{{ ethereum_genesis_fork_version_suffix }}"
+    export DENEB_FORK_EPOCH="10"
```

Comment:
Set to 20 pls, mainly cuz we now have to do a manual deploy. Will write a script today to see if we can script it nicely and use it everywhere.